cmd/deploy/deploy.go

```diff
@@ -195,10 +210,35 @@ func Deploy(ctx context.Context, at analyticsTrackerInterface, ioCtrl *io.IOCont
 				}
 			}
 
+			if shouldRunInPipeline(options) {
```

Comment:
can we add test for the method that checks if pipeline needs to be run? (`shouldRunInPipeline`)